rospy_tutorials/Floats occupancy_grid
geometry_msgs/Point goal_point
nav_msgs/Odometry current_odom
